packages/website/src/pages/_document.tsx

```diff
@@ -1,40 +1,20 @@
-import Document, {Html, Head, Main, NextScript, DocumentContext} from 'next/document';
+import {Head, Html, Main, NextScript} from 'next/document';
 import * as React from 'react';
 
-class MyDocument extends Document {
-    static async getInitialProps(ctx: DocumentContext) {
-        const originalRenderPage = ctx.renderPage;
+const Document: React.FunctionComponent = () => (
+    <Html>
+        <Head>
+            <link rel="stylesheet" href="https://use.typekit.net/wqe4zqp.css" />
```

Comment:
I see a lot of problem with using google font at the moment with GDPR, typekit would not have the same problem?